File: Java/TelefonoUI.java
import java.util.Scanner;

public class TelefonoUI {

  public static void showMenu() {
    int respuesta = 0;

    do {
      TelefonoUtils.imprimir("Bienvenido!");
      System.out.println("Seleciones una opcion:");
      System.out.println("1. Covertir palabra a numero");
      System.out.println("0. Salir");

      System.out.println();
      System.out.print("Ingrese opcion: ");
      Scanner scanner = new Scanner(System.in);

      respuesta = Integer.valueOf(scanner.nextLine());

      switch (respuesta) {
        case 0:
          TelefonoUtils.imprimir("! Se terminó el programa.");
          break;
        case 1:
          TelefonoUtils.imprimir("Convertir palabra a numero");

          System.out.print("Introduzca el texto: ");
          Scanner entrada = new Scanner(System.in);
          String nombre = entrada.nextLine().toLowerCase();

          String[] cadena = nombre.split("");
          String[] num = TelefonoUtils.miSwitch(cadena);
          TelefonoUtils.resultado(num);
          break;
        default:
          TelefonoUtils.imprimir("! Selecciona una opción correcta.");
      }
    } while (respuesta != 0);

  }
}

File: C#/TelefonoUI.cs
using System;

namespace consoleProject
{
  public class TelefonoUI
  {
    public static void mostrarMenu() {
      TelefonoUtils.imprimirTitulo("Bienvenido");

      int userInteface = 0;
      do
      {
        Console.WriteLine("Seleccione una opción");
        Console.WriteLine(" 1. Ingresar palabra");
        Console.WriteLine(" 0. Salir");
        string selectNum = Console.ReadLine().ToString();

        userInteface = 1;
        switch (selectNum)
        {
          case "0":
            userInteface = 0;
            TelefonoUtils.imprimirTitulo("Programa Finalizado");
            break;
          case "1":
            TelefonoUtils.imprimirTitulo("Introduzca el texto");
            string palabra = Console.ReadLine();

            char[] palabraTrim = new char[palabra.Length];
            for (int i = 0; i < palabra.Length; i++)
            {
              palabraTrim[i] = palabra[i];
            }

            String[] numeros = TelefonoUtils.miSwitch(palabraTrim);
            TelefonoUtils.resultado(numeros);
            break;
          default:
            TelefonoUtils.imprimirTitulo("Seleccione una opcion correcta");
            break;
        }

      } while (userInteface != 0);
    }
  }
}
